FrederickMunro/wiley-team-three
# Added ChartJS implementation and Readme

Description:
Rough implementation of ChartJS and creation of frontend portion of Readme.

Testing:
1. Verify Readme looks good from github branch
2. Pull branch, npm run dev, verify that 3 charts load up.
